doc/guides/misc/how_to_doc.md

```diff
@@ -0,0 +1,156 @@
+which means that we both have the control over the content and layout but at the same time have to put more active effort into the UX of the documentation itself.
+
+## How to run the site locally
+
+To run the site locally, you need to have [Node.js](https://nodejs.org/) installed.
+It is highly recommended to use a version manager such as [NVM](https://github.com/nvm-sh/nvm) to manage your Node.js versions.
+
+Once installed you should be able to install the dependencies and run the site locally with the following commands:
+
+```bash title="Running the site locally"
+make dev
+```
+
+This will start a local development server and open the site in your default web browser. The resulting output should look like this:
+
+```bash title="Output of running the site locally"
+(3.12.8) [ann@ann-laptop13 starlight]$ make dev
+Installing starlight...
+
+Starlight installed successfully.
+Starting starlight live server...
+
+> guide@0.0.1 dev
+> astro dev
+
+13:32:10 [types] Generated 0ms
+13:32:10 [content] Syncing content
+13:32:10 [content] Synced content
+
+ astro  v5.6.1 ready in 1490 ms
+
+┃ Local    http://localhost:4321/
+┃ Network  use --host to expose
+
+13:32:10 watching for file changes...
+```
+
+You can then browse the site at the URL shown in the output, which is usually `http://localhost:4321/`.
+
+One big advantage of Starlight is that it supports hot reloading, which means that any changes you make to the content will be reflected in the browser immediately without needing to refresh the page. This makes it easy to see how your changes affect the site in real-time.
+
+You can also use the `make build` command to build the site for production. This will create a static version of the site in the `dist` directory, which can be deployed to any static file hosting service.
+
+```bash title="Building the site for production"
+make build
+```
+
+## How to create a new guide
+
+Now that we understand how to run the site locally, let's look at how to create a new guide.
+The first step is to create a Markdown file within `doc/guides/` with the following structure:
+
+```markdown title="Example guide structure"
+---
+title: My New Guide
+description: This is a description of my new guide
+---
+
+Hello World!
+```
+
+Starlight uses the frontmatter of the Markdown file to generate the navigation and other metadata for the guide. On Github this will appear as a small table at the top of the file so your guide will still be readable in the repository.
+
+Now you can add content to the guide using Markdown. Starlight supports all the standard Markdown features, including headings, lists, links, images, and code blocks. You can also use MDX to embed JavaScript, Components, and other features directly into your Markdown files. This allows you to create interactive components and dynamic content within your guides.
```

Comment:
is there documentation where I can look up on what is possible and how to do it?

Reply:
added